mitmproxy/addons/serverplayback.py

```diff
@@ -166,7 +175,7 @@ def next_flow(self, request):
                 return ret
 
     def configure(self, updated):
-        if not self.configured and ctx.options.server_replay:
+        if not self.configured and not updated:
```

Comment:
This would read all flows form disk whenever any options is changed, so this has unintended side-effects. 

The intended way from within mitmproxy is to call `replay.server`, but that's confusing currently because editing the option is a no-op. For now, the easiest approach is probably to check here if `"server_replay" in updated` and if so, call either `replay.server.file` or `replay.server.stop`.